Makefile

```
@@ -59,10 +56,10 @@ pushversion:
 
 .PHONY: publish
 publish:
-	python setup.py sdist bdist_wheel
+	python -m build --wheel
```

why just wheel, though?
```suggestion
	python -m build
```